central/tlsconfig/tlsconfig_test.go

```diff
@@ -31,13 +31,24 @@ func (s *tlsConfigTestSuite) TestGetAdditionalCAs() {
 
 	additionalCAs, err := GetAdditionalCAs()
 	s.Require().NoError(err)
-	s.Require().Len(additionalCAs, 4, "Could not decode all certs")
+	s.Require().Len(additionalCAs, 5, "Could not decode all certs")
 	s.True(s.isCommonNameInCerts(additionalCAs, "CENTRAL_SERVICE: Central"), "Could not find cert from multiple crt file")
 
 	// non .crt files should be ignored
 	s.FileExists("testdata/foo.txt")
 }
 
+func (s *tlsConfigTestSuite) TestGetAdditionalCAFilePaths() {
+	s.T().Setenv("ROX_MTLS_ADDITIONAL_CA_DIR", "testdata")
+	filePaths, err := GetAdditionalCAFilePaths()
+	s.NoError(err)
```

Comment:
Using `require` stops the test execution. This can be used here because all assertions after the `NoError` would fail in case of an error.

```suggestion
s.Require().NoError(err)
```